tests/regression_tests/ncrystal/test.py

```diff
@@ -1,5 +1,6 @@
 from math import pi
-
+import filecmp
+from difflib import unified_diff
```

Comment:
Our Python code follows the PEP8 style guidelines (see [this section](https://peps.python.org/pep-0008/#imports) on module imports). Here, you just need an extra line to separate the standard library and third-party imports:
```suggestion
import filecmp
from difflib import unified_diff

```